src/components/Logo/style.js

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+export const Svg = styled.svg`
+  margin-left: 20px;
```

Comment:
This css rule is too specific. Such rules should be define on higher level of components e.g. movie page or header.